### PR TITLE
18GA: Add some missing features

### PR DESCRIPTION
**Pull request by perwestling**
Has some problems so do not merge this yet.

### CODE REVIEW

File: lib/engine/game/g_18_ga.rb [hunk 1]
```diff
@@ -27,17 +29,17 @@ def setup
         setup_company_price_50_to_150_percent
 
         # Place neutral tokens in the off board cities
-        neutral = Corporation.new(
+        @neutral = Corporation.new(
```
Comment (tobymao) [inline lib/engine/game/g_18_ga.rb:32]: why make these instance variables? you never use them

File: lib/engine/train.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ class Train
     attr_accessor :obsolete, :operated, :events
     attr_reader :available_on, :name, :distance, :discount, :obsolete_on,
                 :rusts_on, :rusted, :sym, :variant, :variants
-    attr_writer :buyable
+    attr_writer :buyable, :index
```
Comment (tobymao) [inline lib/engine/train.rb:13]: what's the point of index?

Comment (perwestling) [reply lib/engine/train.rb:13]: When adding the new 2 train I do that by cloning another 2 train, and then I set index so it wont collide with another 2 train.

File: lib/engine/game/g_18_ga.rb (hunk 1 above)
Comment (perwestling) [reply lib/engine/game/g_18_ga.rb:32]: Removed

**Comment by perwestling** [comment]
Have split this into smaller PRs.

**Pull request closed without merging**